web/blog/2022-05-16-vast-v2.0/README.md

```diff
@@ -2,6 +2,7 @@
 title: VAST v2.0
 description: VAST v2.0 - Smarter Query Scheduling & Tunable Filters
 authors: dominiklohmann
+slug: 2022-05-16/vast-v2.0
```

Comment:
Why does this even need to be set explicitly? I thought the directory name defines the slug, no?

Reply:
The slug was `2022-05-16/vast-v2.0/README` for this particular example if I didn't set it explicitly.